shared/components/DemoApp/AsyncPostsRoute/Post/Post.js

```diff
@@ -69,12 +67,23 @@ function Post() {
 //       // fetches the post.
 //       return fetchPost(id);
 //     },
-//     {
-//       // Any time the post id changes we need to trigger the work.
-//       shouldWorkAgain: (prevProps, nextProps) =>
-//         prevProps.params.id !== nextProps.params.id,
-//     },
-//   ),
+//     // {
+//     //   // Any time the post id changes we need to trigger the work.
+//     //   shouldWorkAgain: (prevProps, nextProps) =>
+//     //     prevProps.params.id !== nextProps.params.id,
+//     // },
+//   }),
 // )(Post);
 
-export default Post;
+export default withJob({
+  work: (props) => {
+    const { id } = props.match.params;
+    return fetch(`http://jsonplaceholder.typicode.com/posts/${id}`).then(r => r.json());
```

Comment:
The fetch function is only defined in the browser causing the job to failed on the server side. What I don't understand is why there is nothing in the server console saying the function doesn't exist.